components/LoginButton.tsx

```diff
@@ -1,7 +1,7 @@
 import { signIn } from "next-auth/react"
 
 export function LoginButton() {
-  const handleClick = () => signIn("github")
+  const handleClick = async () => await signIn("github", { redirect: false })
```

Comment:
sign-in redirects are only available to credentials and email providers, not OAuth.
docs: https://next-auth.js.org/getting-started/client#using-the-redirect-false-option

Reply:
Thanks for correcting, although I would have liked the page not to reload when logging in.